docs/sections/user_guide/yaml/components/mpas.rst

```diff
@@ -17,15 +17,20 @@ An MPAS build provides prototype versions of certain required runtime files. Her
 UW YAML for the ``mpas:`` Block
 -------------------------------
 
+domain:
+^^^^^^^
+
+Accepted values are ``global`` and ``regional``.
+
 execution:
 ^^^^^^^^^^
 
 See :ref:`this page <execution_yaml>` for details.
 
-boundary_conditions:
-^^^^^^^^^^^^^^^^^^^^
+lateral_boundary_conditions:
```

Comment:
This was a copy/paste issue with MPASInit docs, where MPASInit combines initial and lateral boundary conditions, and MPAS does not.